.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Issues This PR Fixes
+Fixes #NNNN
+Fixes #NNNN
+
+## Related PRs
+List related PRs against other branches e.g. for backporting features/bugfixes
+to previous release branches:
+
+Repo/Branch | PR
+------ | ------
+some_other_PR | [link]()
+
+
+## Todos
+- [ ] Tested and working on development environment
+- [ ] Unit tests (if appropriate)
+- [ ] Added/Updated all related documentation. Add [link]() if different from this PR
+- [ ] DevOps Support needed e.g. create Runscope API test if new endpoint added or
+      update deployment docs. Create a ticket and add [link]()
+
+## Deployment Notes
+Notes about how to deploy this work. For example, running a migration against the production DB.
```

Comment:
I do not understand the relevance of this example with what this repository is about.

Reply:
good point will update